core/templates/dev/head/pages/exploration_editor/translation_tab/StateTranslationDirective.js

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the state translation.
+ */
+
+oppia.directive('stateTranslation', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration_editor/translation_tab/' +
+        'state_translation_directive.html'),
+      controller: [
+        '$scope', '$filter', 'ExplorationStatesService', 'EditorStateService',
+        function(
+            $scope, $filter, ExplorationStatesService, EditorStateService) {
+          // Define tab constants.
+          $scope.TAB_ID_CONTENT = 'content';
+          $scope.TAB_ID_FEEDBACK = 'feedback';
+          $scope.TAB_ID_HINTS = 'hints';
+          $scope.TAB_ID_SOLUTION = 'solution';
+
+          // Activates Content tab by default.
+          $scope.ACTIVATED_TAB_ID = $scope.TAB_ID_CONTENT;
```

Comment:
Maybe this should be camel case instead (activatedTabId), since it's not a constant?

Reply:
Done!